pkg/hubble/metrics/api/api.go

```diff
@@ -146,3 +128,101 @@ var registry = NewRegistry(
 func DefaultRegistry() *Registry {
 	return registry
 }
+
+// FlowFilters is a slice of filters to include or exclude flows.
+type FlowFilters []*pb.FlowFilter
+
+type ContextValues []string
+
+// ContextOptions is a structure used for configuring Hubble metrics context options
+type ContextOptionConfig struct {
+	Name   string        `json:"name,omitempty" yaml:"name,omitempty"`
+	Values ContextValues `json:"values,omitempty" yaml:"values,omitempty"`
+}
+
+type ContextOptionConfigs []*ContextOptionConfig
```

Comment:
Not sure you need a type here, can just use `[]*ContextOptionConfig` where needed.